app/views/thread_mailer/_motion.html.haml

```diff
@@ -13,11 +13,11 @@
         - if motion.closed?
           %b= t(:closed) + " " + time_formatted_relative_to_age(motion.closing_at)
           - if motion.outcome.present?
-            = render_email_plaintext motion.outcome
```

Comment:
looking over this.. I think you should be using render_rich_text(text, true) I think render_email_markdown is just for yesterday on loomio